src/test/java/com/passbook/sparkeighteen/controller/UserControllerTest.java

```diff
@@ -96,4 +96,28 @@ public void validRequest_errorResponse_LoginSuccessfull() throws Exception {
         assertEquals(response.getStatusCode(), HttpStatus.OK);
 
     }
+
+    @Test
+    public void validRequest_successfulResponse_DeleteSuccessfull() throws Exception {
```

Comment:
No test cases for negative cases why?

Reply:
there are negative validRequest_errorResponse_DeleteUnSuccessfull() which is negative.